DEVELOPMENT.md

```diff
@@ -0,0 +1,229 @@
+# Development
+
+M3 is a very active, large codebase. This guide is intended for all potential
```

Comment:
```suggestion
M3 is an active, large codebase. This guide is intended for all potential
```

I need to create a proper style guide etc, but words like "very" are often not needed and add nothing